utils/auditbasicparams.go

```diff
@@ -40,6 +40,8 @@ type AuditParams interface {
 	Exclusions() []string
 	SetIsRecursiveScan(isRecursiveScan bool) *AuditBasicParams
 	IsRecursiveScan() bool
+	SetDownloadUrls(urlsMap map[string]string)
+	GetDownloadUrls() map[string]string
```

Comment:
I don't think this should be added as parameters.

Reply:
We can discuss it, I answered on it in https://github.com/jfrog/jfrog-cli-security/pull/39#discussion_r1542933177

Reply:
Fixed